__tests__/seed.test.js

```diff
@@ -41,86 +41,138 @@ describe("/api/categories", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=review_id")
+        .expect(200)
+        .then((res) => {
+          const reviews = res.body.reviews;
 
-  test("200: responds with an array of objects correctly sorted by custom sort query, in descending order by default", () => {
-    return request(app)
-      .get("/api/reviews?sort_by=review_id")
-      .expect(200)
-      .then((res) => {
-        const reviews = res.body.reviews;
+          expect(reviews).toBeSortedBy("review_id", { descending: true });
+        });
+    });
 
-        expect(reviews).toBeSortedBy("review_id", { descending: true });
-      });
+    test("200: responds with an array of objects sorted by custom query, in ascending order", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=title&order=asc")
+        .expect(200)
+        .then((res) => {
+          const reviews = res.body.reviews;
+
+          expect(reviews).toBeSortedBy("title", { ascending: true });
+        });
+    });
   });
 
-  test("200: responds with an array of objects sorted by custom query, in ascending order", () => {
-    return request(app)
-      .get("/api/reviews?sort_by=title&order=asc")
-      .expect(200)
-      .then((res) => {
-        const reviews = res.body.reviews;
+  describe("Errors", () => {
+    test("400: responds with a 400 error if the path does not exist", () => {
+      return request(app)
+        .get("/api/reviewz")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad Request");
+        });
+    });
 
-        expect(reviews).toBeSortedBy("title", { ascending: true });
-      });
+    test("404: responds with a 404 error if given an invalid order query", () => {
+      return request(app)
+        .get("/api/reviews?order=asdc")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Not Found");
+        });
+    });
+
+    test("404: responds with a 404 error if given an invalid sort query", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=name")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Not Found");
+        });
+    });
   });
+});
 
-  test("404: responds with a 404 error if given an invalid order query", () => {
-    return request(app)
-      .get("/api/reviews?order=asdc")
-      .expect(404)
-      .then(({ res }) => {
-        expect(res.statusMessage).toBe("Not Found");
-      });
+describe("/api/reviews/:review_id", () => {
+  describe("Happy path", () => {
+    test("200: responds with a review object that corresponds to the submitted ID", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((res) => {
+          expect(res.body.review).toMatchObject({
+            review_id: 1,
+            review_body: expect.any(String),
+            designer: expect.any(String),
```

Comment:
It would be preferable to expect the exact values associated with review 1 here instead of doing expect.any(value)